source/manual/redirect-routes.html.md

```diff
@@ -58,6 +58,82 @@ and using non-default values for the `segments_mode`.
 [short-url-manager]: https://short-url-manager.publishing.service.gov.uk
 [short-url-manager-permissions]: https://github.com/alphagov/short-url-manager/#permissions
 
+### Removing a route created in the Short URL Manager
+
+First find the `content_id` of the URL that needs to be removed. This can be found by using the Content Store API: `www.gov.uk/api/content/<path goes here>`.
+
+For example, finding the `content_id` for the page about [keeping a pet pig or micropig](https://www.gov.uk/guidance/keeping-a-pet-pig-or-micropig) is found by going to [`www.gov.uk/api/content/guidance/keeping-a-pet-pig-or-micropig`](https://www.gov.uk/api/content/guidance/keeping-a-pet-pig-or-micropig). The `content_id` should be a UUID; for example: `602144f8-7631-11e4-a3cb-005056011aef`.
+
+The content_id is with a rake task that removes the shortlink. The [Short URL Manager documentation](https://docs.publishing.service.gov.uk/apps/short-url-manager.html#run-a-rake-task) has handy links for running the rake tasks in the different environments.
+
+The rake task for removing a redirect is:
+
+```
+redirect:destroy["<uuid goes here>"]
```

Comment:
The dev docs now have a helper that will produce links to the 'Run rake task' job in Jenkins for all the environments.  It might be easier for those reading the documentation to have a link they can click?
```suggestion
<%= RunRakeTask.links("asset-manager", "redirect:destroy[<uuid goes here>]") %>```

Reply:
That is a nifty helper - thank you for sharing. I've added it in (and also discovered that it really doesn't like `<uuid goes here>` as that then closes one of the HTML tags...)